### ISSUE TITLE
Sign error, wrong status   "status":"invalid"

### ISSUE THREAD

**Issue by bineyond** [acmesh-official/acme.sh#3703]
此域名下的所有证书申请都失败，一样的错误

------------------

Debug log
-----------------

```
[2021年 9月20日 星期一 03时46分35秒 CST] Polling order status: https://acme.zerossl.com/v2/DV90/order/bpvs1spJXD5lhYQRiYknow
[2021年 9月20日 星期一 03时46分35秒 CST] url='https://acme.zerossl.com/v2/DV90/order/bpvs1spJXD5lhYQRiYknow'
[2021年 9月20日 星期一 03时46分35秒 CST] payload
[2021年 9月20日 星期一 03时46分35秒 CST] Retrying post
[2021年 9月20日 星期一 03时46分35秒 CST] POST
[2021年 9月20日 星期一 03时46分35秒 CST] _post_url='https://acme.zerossl.com/v2/DV90/order/bpvs1spJXD5lhYQRiYknow'
[2021年 9月20日 星期一 03时46分35秒 CST] _CURL='curl --silent --dump-header /Users/ubin/.acme.sh/http.header  -L  -g '
[2021年 9月20日 星期一 03时46分36秒 CST] _ret='0'
[2021年 9月20日 星期一 03时46分36秒 CST] _hcode='0'
[2021年 9月20日 星期一 03时46分36秒 CST] code='200'
[2021年 9月20日 星期一 03时46分36秒 CST] Sign error, wrong status
[2021年 9月20日 星期一 03时46分36秒 CST] {"status":"invalid","expires":"2021-12-18T16:31:37Z","identifiers":[{"type":"dns","value":"*.tun.ubin.top"}],"authorizations":["https://acme.zerossl.com/v2/DV90/authz/uNr6MJ3W58pFFCaOa_GxwQ"],"finalize":"https://acme.zerossl.com/v2/DV90/order/bpvs1spJXD5lhYQRiYknow/finalize"}
[2021年 9月20日 星期一 03时46分36秒 CST] _on_issue_err
[2021年 9月20日 星期一 03时46分36秒 CST] Please add '--debug' or '--log' to check more details.
[2021年 9月20日 星期一 03时46分36秒 CST] See: https://github.com/acmesh-official/acme.sh/wiki/How-to-debug-acme.sh
[2021年 9月20日 星期一 03时46分36秒 CST] The dns manual mode can not renew automatically, you must issue it again manually. You'd better use the other modes instead.
[2021年 9月20日 星期一 03时46分36秒 CST] Diagnosis versions:
```

**Comment by bineyond** [comment]
another domain no this error

**Comment by Neilpang** [comment]
```
[2021年 9月20日 星期一 03时46分36秒 CST] See: https://github.com/acmesh-official/acme.sh/wiki/How-to-debug-acme.sh
```

**Comment by bineyond** [comment]
log 
https://gist.github.com/bineyond/e0f754c7ca3f03fdfe5a72bb3e8af4c8 

![nCcjqmRvLyzhe8E](https://i.loli.net/2021/09/20/nCcjqmRvLyzhe8E.png)

![dQKigaNV26GyMlC](https://i.loli.net/2021/09/20/dQKigaNV26GyMlC.png)

![EJLaXyCTsY4fZom](https://i.loli.net/2021/09/20/EJLaXyCTsY4fZom.png)

**Comment by bineyond** [comment]
@Neilpang Can you help me figure out what’s causing this?

**Comment by TonyCSB** [comment]
I have found the same problem. Upon further investigation, I found the issue. In my case, it was CAA record missing. The domain I am trying to sign `$domain` has a CNAME record. For whatever reason, when trying to get a CAA record under the `$domain`, the request always timeout. Hence, the CA could not issue the certificate. My workaround is to temporarily disable the CNAME record and issue the certificate, then enable the CNAME record after the certificate is renewed.
